.github/workflows/unit-testing.yml

```diff
@@ -0,0 +1,63 @@
+name: unit-testing
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  unit-testing:
+    runs-on: ubuntu-latest
+    outputs:
+      has_detections: ${{ steps.check_detections.outputs.has_detections }}
+    if: "!contains(github.ref, 'refs/tags/')" #don't run on tags - future steps won't run either since they depend on this job
+    # needs: [validate-tag-if-present, quit-for-dependabot]
+    steps: 
+        - name: Check out the repository code
+          uses: actions/checkout@v4
+          with:
+            ref: develop
+
+        - uses: actions/setup-python@v5
+          with:
+            python-version: '3.11' #Available versions here - https://github.com/actions/python-versions/releases  easy to change/make a matrix/use pypy
+            architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
+
+        - name: Install System Packages and contentctl 
+          run: |
+            sudo apt update -qq
```

Comment:
Now that we are doing the processing with python i think we can remove jq?

Reply:
removed